cmd/kube-controller-manager/app/core.go

```diff
@@ -290,8 +291,11 @@ func startAttachDetachController(ctx context.Context, controllerContext Controll
 		return nil, true, err
 	}
 
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "volume controller"))
+	logger := klog.FromContext(ctx)
```

Comment:
This is a bit odd sequence. This is simpler:
```
logger := klog.LoggerWithName(klog.FromContext(ctx), "volume controller")
ctx = klog.NewContext(ctx, logger)
```

Reply:
done